controller/konnect/reconciler_kongplugin_rbac.go

```diff
@@ -6,3 +6,4 @@ package konnect
 //+kubebuilder:rbac:groups=configuration.konghq.com,resources=kongservices,verbs=get;list;watch;update;patch
 //+kubebuilder:rbac:groups=configuration.konghq.com,resources=kongroutes,verbs=get;list;watch;update;patch
 //+kubebuilder:rbac:groups=configuration.konghq.com,resources=kongpluginbindings,verbs=get;list;create;watch;update;patch;delete
+//+kubebuilder:rbac:groups=configuration.konghq.com,resources=kongpluginbindings/status,verbs=get;update;patch
```

Comment:
There is no such thing as get for object's status

```suggestion
//+kubebuilder:rbac:groups=configuration.konghq.com,resources=kongpluginbindings/status,verbs=update;patch
```

Reply:
Other resources are also using the set of verbs. Looks like `get` is not required so I removed it. Should we also remove the `get` for status of other types?

Reply:
Yup, that had to be a mistake IMO when adding RBAC rules.